From 6decd49761a30fb9560294af3e3b1972ee8efe05 Mon Sep 17 00:00:00 2001
From: Peter Foley <pefoley@google.com>
Date: Fri, 28 May 2021 13:31:11 -0400
Subject: [PATCH] Add a missing include

e.g.
third_party/ijar/mapped_file_unix.cc: In constructor 'devtools_ijar::MappedOutputFile::MappedOutputFile(const char*, size_t)':
third_party/ijar/mapped_file_unix.cc:115:21: error: 'numeric_limits' is not a member of 'std'
  115 |                std::numeric_limits<size_t>::max());
---
 third_party/ijar/mapped_file_unix.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/ijar/mapped_file_unix.cc b/third_party/ijar/mapped_file_unix.cc
index 6e3a90871844..030e9ca59fa4 100644
--- a/third_party/ijar/mapped_file_unix.cc
+++ b/third_party/ijar/mapped_file_unix.cc
@@ -19,6 +19,7 @@
 #include <sys/mman.h>
 
 #include <algorithm>
+#include <limits>
 
 #include "third_party/ijar/mapped_file.h"
 
Index: bazel/third_party/ijar/zlib_client.cc
===================================================================
--- bazel.orig/third_party/ijar/zlib_client.cc
+++ bazel/third_party/ijar/zlib_client.cc
@@ -15,6 +15,7 @@
 #include <stdlib.h>
 #include <algorithm>
 #include <cstdio>
+#include <limits>

 #include "third_party/ijar/common.h"
 #include "third_party/ijar/zlib_client.h"
